src/components/side-bar/side-bar.js

```diff
@@ -30,9 +30,9 @@ const SideBar = ({ ruleTotalNumber, location }) => {
       </section>
 
       <section id="widget">
-        <Widget
+        <RulesWidget
           token={process.env.GITHUB_API_PAT}
-          location={location}
+          location="https://www.ssw.com.au/rules"
```

Comment:
To avoid hardcoding this, you can use the `siteUrl` in the site-config.js file

https://github.com/SSWConsulting/SSW.Rules/blob/52ab0a8844d6e26515f1d9ed0eb75bf1056cab68/site-config.js#L7-L8